src/Atoms/ButtonsExercises/ButtonsExercises.js

```diff
@@ -0,0 +1,38 @@
+import ButtonExercisesStyled from './ButtonExercisesStyled';
+import { FaChevronLeft } from 'react-icons/fa';
+import propTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+
+const ButtonExercises = (props) => {
+  const { endingButton, goBack, text } = props;
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    if (goBack) {
+      navigate(-1);
```

Comment:
Will there always be a case of going back? What if the page has been loaded from link?

Reply:
I didn't consider this case - I have to ask our linking specialist :)